src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -59,10 +59,10 @@ public function renderAttributesAction()
      */
     public function renderAttributeValueFormsAction(Request $request)
     {
-        $attributeRepository = $this->get('sylius.repository.product_attribute');
+        $attributeRepository = $this->get('sylius.repository.'.$this->config->getResourceName());
         $forms = array();
 
-        $choices = ($request->query->has('sylius_product_attribute_choice')) ? $request->query->get('sylius_product_attribute_choice') : array();
+        $choices = ($request->query->has(sprintf('sylius_%s_choice', $this->config->getResourceName()))) ? $request->query->get(sprintf('sylius_%s_choice', $this->config->getResourceName())) : array();
```

Comment:
micro-optimization but you could set the string before using it twice (it will make the line shorter too).